src/html.ml

```diff
@@ -1,5 +1,29 @@
 open Ast
 
+type printer =
+  {
+    start: printer          -> Buffer.t -> inline block list                                             -> unit;
+    finish: printer         -> Buffer.t                                                                  -> unit;
+    block: printer          -> Buffer.t -> inline block                                                  -> unit;
+    paragraph: printer      -> Buffer.t -> inline                                                        -> unit;
+    blockquote: printer     -> Buffer.t -> inline block list                                             -> unit;
+    list: printer           -> Buffer.t -> list_kind * list_style * inline block list list               -> unit;
```

Comment:
Since these are functions we could use multiple arguments instead of tuples. Even better, why not define suitable record types and use them both for the types `block` and `inline` and here?

Reply:
Indeed, I kept the tuples without thinking about it, thx